### PR TITLE
Refactor to modular approach

### PR DESCRIPTION
**Pull request by minottic**
This is what I would do to make it more "composable", for issue #35 .

I have split it into commits with a prefix for readability, if we are fine with this approach I will cherry-pick and open dedicated PRs. 

There are README files per service, but I came to think that a detailed setup description per service should be in every component repo, on the home page and these READMEs will point to them (similar to what the `frontend` README here does). Then what the READMEs hare have left to do is describe their role in scicatlive and what they connect to. 

As you can see there isn't yet the need of `profiles` but I think we will add them going forward

### CODE REVIEW

**Review by fpotier — COMMENTED** [review]
I think we could get rid of the `mongoseed` by placing the script in `MONGODB_INITSCRIPTS_DIR` (See 'Read-only environment variables' section https://hub.docker.com/r/bitnami/mongodb)

**Comment by minottic** [comment]
ah nice! I think I tried it in the past, but the problem was that the INIT is run before the DB is up, but it's worth trying again as I don't remember well. I will open an issue for that and we can do it in another PR

File: README.md [hunk 1]
```diff
@@ -15,41 +15,61 @@ Files for running SciCat with docker-compose.
...
+For example, one could decide to only run the `backend` by running:
 
-| Username         | Password    | Usage                        |
-| ---------------- | ----------- | ---------------------------- |
-| admin            | 2jf70TPNZsS | Admin                        |
-| ingestor         | aman        | Ingest datasets              |
-| archiveManager   | aman        | Manage archiving of datasets |
-| proposalIngestor | aman        | Ingest proposals             |
+```sh
+docker-compose up -d backend
```
Comment (LAShemilt) [inline README.md:44]: do the traefik labels have an affect here? Also the dependency on mongo ? I can't see these being used in the compose for the backend.

**Review by LAShemilt — COMMENTED** [review]
I really like this modular approach it is very extensible. I think we need to think carefully about how the dependencies are going to work.

**Comment by minottic** [comment]
hi Laura, thanks for the comment! yes, the dependencies are working fine, almost everything is already on master, there is an upcoming README change PR coming!

**Comment by minottic** [comment]
closed by linke PRs/issues

**Pull request closed without merging**